### DISCUSSION TITLE
City Name

### DISCUSSION THREAD

**Discussion by togotom** [JaouherK/streamDeck-weatherPlugin#8] [Ideas]
Hello Jaouher
Is it also possible to hide the City name I only want to show the title name.

**Comment by JaouherK** [comment]
Hi @togotom , Yeah I am working already on it 😄 I will provide the possibility to show/hide the city name

**Reply by togotom** [reply]
that's good to hear

**Reply by JaouherK** [reply]
the new version has been submitted to Elgato team. Waiting for them to validate and submit to the store
